data/hERG_Karim_et_al/meta.yaml

```diff
@@ -0,0 +1,40 @@
+name: hERG_Karim_et_al
+description: A integrated Ether-a-go-go-related gene (hERG) dataset consisting of
+  molecular structures labelled as hERG (<10uM) and non-hERG (>=10uM) blockers in
+  the form of SMILES strings was obtained from the DeepHIT, the BindingDB database,
+  ChEMBL bioactivity database, and other literature.
+targets:
+- id: hERG_blocker
+  description: whether it blocks (1, <10uM) or not blocks (0, >=10uM)
+  units: activity
+  type: categorical
+  names:
+  - hERG blocker
+  - hERG active compound
+  - hERG blocker
+  - hERG active compound <10uM
+  - "Human ether-\xE0-go-go related gene (hERG) blocker"
+  - "Activity against Human ether-\xE0-go-go related gene (hERG)"
```

Comment:
can we remove \xE0 and write our the acronym?

Reply:
> Can we remove \xE0 and write the acronym?

Sorry for being late. Yeah, I will proceed in that.